PyInstaller/building/utils.py

```diff
@@ -473,3 +475,93 @@ def format_binaries_and_datas(binaries_or_datas, workingdir=None):
                                 os.path.normpath(src_file)))
 
     return toc_datas
+
+
+def _load_code(modname, filename):
```

Comment:
You tell me - you wrote it ;). I only moved it here to make it available to CArchiveWriter. Previously it was a method on PYZ that was only called to fill in gaps in the `code_dict` when a module's code wasn't already available through the modulegraph.

Since there are so few of them, I decided it was simpler to call this code for the startup scripts than to try and go back to the modulegraph for them, especially since I'd need to do it anyway "in the rare case if outXX-Analysis.toc exists, but outXX-PYZ.toc does not"